README.md

```diff
@@ -40,42 +40,125 @@ Designed for extensibility, scalability, and seamless containerized deployment,
 
 ### Download and Run WSO2 Thunder
 
+You can run WSO2 Thunder either by downloading the release artifact or using the official Docker image.
+
+#### Option 1: Run from Release Artifact
+
 Follow these steps to download the latest release of WSO2 Thunder and run it locally.
 
-#### Step 1: Download the distribution from the latest release
+1. **Download the distribution from the latest release**
 
-Download `thunder_<os>_<arch>-<version>.zip` from the [latest release](https://github.com/asgardeo/thunder/releases/latest) for your operating system and architecture.
+    Download `thunder_<os>_<arch>-<version>.zip` from the [latest release](https://github.com/asgardeo/thunder/releases/latest) for your operating system and architecture.
 
-For example, if you are using a MacOS machine with a Apple Silicon (ARM64) processor, you would download `thunder_macos_arm64-<version>.zip`.
+    For example, if you are using a MacOS machine with a Apple Silicon (ARM64) processor, you would download `thunder_macos_arm64-<version>.zip`.
 
-#### Step 2: Unzip and start the product
+2. **Unzip the product**
 
-- Unzip the downloaded file using the following command:
+    Unzip the downloaded file using the following command:
 
-  ```bash
-  unzip thunder-<os>_<arch>-<version>.zip
-  ```
+    ```bash
+    unzip thunder-<os>_<arch>-<version>.zip
+    ```
 
-- Navigate to the unzipped directory:
+    Navigate to the unzipped directory:
 
-  ```bash
-  cd thunder-<os>_<arch>-<version>/
-  ```
+    ```bash
+    cd thunder-<os>_<arch>-<version>/
+    ```
+
+3. **Start the product**
 
-- Start the product using the following command:
+    Start the product using the following command:
 
-  - If you are using a Linux or macOS machine:
+    If you are using a Linux or macOS machine:
 
     ```bash
     bash start.sh
     ```
 
-  - If you are using a Windows machine:
+    If you are using a Windows machine:
 
     ```bash
     start.bat
     ```
 
+4. **(Optional) Start with HTTP (no TLS)**
+
+    If you want to run Thunder with HTTP (not recommended for production), update the `deployment.yaml` file in the `repository/conf` directory as follows:
+
+    ```yaml
+    server:
+      hostname: "localhost"
+      port: 8090
+      http_only: true
+    ```
+
+    Restart the server using the command mentioned in step 3.
+
+#### Option 2: Run with Docker
+
+Follow these steps to run WSO2 Thunder using Docker.
+
+1. **Pull the Docker image**
+
+    ```bash
+    docker pull ghcr.io/asgardeo/thunder:latest
+    ```
+
+2. **Prepare the configuration file**
+
+    Create a `deployment.yaml` file in your working directory and copy the content from [deployment.yaml](https://github.com/asgardeo/thunder/blob/main/backend/cmd/server/repository/conf/deployment.yaml).
+
+    Update the server configs as below:
+
+    ```yaml
+    server:
+      hostname: 0.0.0.0
+      port: 8090
+    ```
```

Comment:
Can we avoid this step?

Reply:
As per the offline discussion, added a docker file instruction to modify the server hostname to `0.0.0.0`. This eliminates the need to mount a config file.

0e77ce150e33c8ddbf8d5f176c697e9b520c2251